lib/actions/siriusXM.js

```diff
@@ -87,23 +70,20 @@ function siriusXM(player, values) {
   } else {
   // Play the specified SiriusXM channel or station
 
-    return getAccountId(player)
-    .then(() => {
-      if (accountId != '') {
-        var searchVal = values[0];
-        var fuzzy = new Fuse(channels, { keys: ["channelNum", "title"] });
+    if (settings.siriusxm.accountid != '') {
```

Comment:
I get a crash if I don't have a settings.json file, so I think you should change this to the following:

if (typeof(settings.siriusxm) != 'undefined' && settings.siriusxm.accountid != '') {